### ISSUE TITLE
Broken at Ubuntu 14.04

### ISSUE THREAD

**Issue by Rafael-MoNa** [slgobinath/SafeEyes#132]
Hi,

This broke at Ubuntu 14.04 
```
$ safeeyes
Traceback (most recent call last):
  File "/usr/bin/safeeyes", line 9, in <module>
    load_entry_point('safeeyes==1.2.0a10', 'console_scripts', 'safeeyes')()
  File "/usr/lib/python3/dist-packages/safeeyes/__main__.py", line 236, in main
    if not running():
  File "/usr/lib/python3/dist-packages/safeeyes/__main__.py", line 208, in running
    cmd_line = proc.cmdline()
TypeError: 'list' object is not callable
```
Not sure why but after the last update the path also changed from /opt/safeeyes to `env GDK_BACKEND=x11 safeeyes`.

I've an NVidia card in case that makes a difference.

**Comment by slgobinath** [comment]
Hi,
Please upgrade the psutil using the following command.
```
sudo apt-get install python3-pip
sudo pip3 install --upgrade psutil
```

The problem is version mismatch of `psutil`. I will fix it in the deb build soon.

Thanks for reporting this issue.

**Comment by slgobinath** [comment]
Fixed in v1.2.1. Please upgrade the Safe Eyes.
